src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
-        System.out.println("Hello from\n" + logo);
+        System.out.println("\nHello from\n" + logo);
+
+        System.out.println(boundary+"\nHello! I'm Duke\n" + "What can I do for you?\n" +boundary);
+
+
+        while(!EndConvo){
+            if(!scan.hasNextLine()){
+                return;
+            } else{
+                String CommandTxt = scan.nextLine();
+                String[] Message = CommandTxt.split(" ", 2);
+
+                if (CommandTxt.equals("bye")) {
+                    EndConvo = true;
+                } else if (CommandTxt.equals("list")) {
+                    System.out.println(boundary + "\n");
+
+                    for (int i = 0; i < taskno; i++) {
+                        System.out.println(String.format("%d", index) + "." + item[i]);
+                        index++;
+                    }
+                    System.out.println(boundary);
+                    index = 1;
+                } else if (Message[0].equals("done")) {
+                    int donetaskno = Integer.parseInt(Message[1]);
+                    item[donetaskno - 1].markAsDone();
+                    System.out.println("Nice! I've marked this task as done: " + "\n");
+                    System.out.println("\t[" + item[donetaskno - 1].getStatusIcon() + "] " + item[donetaskno - 1].description);
+                    System.out.println(boundary);
+                } else if (Message[0].equals("todo")) {
+                    item[taskno] = new ToDo(Message[1]);
+                    System.out.println(boundary + "\nGot it. I've added this task: " + System.lineSeparator()
+                            + "  " + item[taskno]);
+                    System.out.println("Now you have " + (taskno + 1) + " task in the list\n" + boundary);
+                    taskno++;
+                } else if (Message[0].equals("deadline")) {
+                    String[] DeadlineMessage = Message[1].split("/by ", 2);
+                    item[taskno] = new Deadline(DeadlineMessage[0], DeadlineMessage[1]);
+                    System.out.println(boundary + "\nGot it. I've added this task: " + System.lineSeparator()
+                            + "  " + item[taskno]);
+                    System.out.println("Now you have " + (taskno + 1) + " task in the list\n" + boundary);
+                    taskno++;
+                } else if (Message[0].equals("event")) {
+                    String[] EventMessage = Message[1].split("/at", 2);
+                    item[taskno] = new Events(EventMessage[0], EventMessage[1]);
+                    System.out.println(boundary + "\nGot it. I've added this task: " + System.lineSeparator()
+                            + "  " + item[taskno]);
+                    System.out.println("Now you have " + (taskno + 1) + " task in the list\n" + boundary);
+                    taskno++;
+                } else {
+                    System.out.println("Command not recognized, Please enter correct command\n" + boundary);
```

Comment:
you can make it with just one System.out.println("...\n"). perhaps, use "\n" after each sentence